phys2bids/physio_obj.py

```diff
@@ -228,6 +228,21 @@ class BlueprintInput():
     def __init__(self, timeseries, freq, ch_name, units, trigger_idx,
                  num_timepoints_found=None, thr=None, time_offset=0):
         """Initialise BlueprintInput (see class docstring)."""
+        trigger_names_list = ["trig", "trigger", "TRIGGER", "Trigger"]
```

Comment:
This could become a constant, initialised at the beginning of the file - it would also be easier to find if we need to update it.

Another thing that could be done is to:
- have only lowercases/uppercases (so you don't have to specify all the variations of `trigger`, for instance, like we're doing for the units of measure),
- order the items from the longest to the shortest (so that first `trigger` is searched, then `trigg`, ... then, if we want, `t`),
- implement a partial match (like @tsalo is suggesting further).

In case we only use the lowercase or uppercase, we can rename the channel `trigger` (since that is the suggested BIDS name).

This would reduce the length of the list and support more possibilities.